ocs_ci/ocs/ui/ols_ui.py

```
@@ -0,0 +1,201 @@
+        for xpath in (self.SEND_BUTTON_TYPE_XPATH, self.SEND_BUTTON_ARIA_XPATH):
+            try:
+                self._wait_until_elements_exist(
+                    xpath, timeout=constants.OLS_UI_SEND_BUTTON_TIMEOUT_SEC
+                )
+                self.do_click_by_xpath(xpath)
+                return
+            except TimeoutException:
+                continue
+        raise TimeoutException("No send button found for OLS chat")
+
+    def _wait_for_answer(self, timeout=None):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_wait_for_answer`

(ANN202)